ref/appliance-command-reference/monitor.md

```diff
@@ -0,0 +1,50 @@
+The `monitor` command allows managing the monitoring dashboard. You can expose or unexpose the monitoring dashboard using the subcommands. By default, the monitoring dashboard is exposed.
+
+## Usage
+
+    anbox-cloud-appliance monitor <subcommand>
+
+## Subcommands
+
+### `account`
+Manages accounts that can access the monitoring dashboard.
+
+#### Usage
+
+    anbox-cloud-appliance monitor account <subcommand>
+
+#### Subcommands
+
+##### `create`
+Creates a new account to access the monitoring dashboard. The monitoring dashboard is configured with basic authentication and hence you need to create an account to access it.
+
+###### Usage
+
+    anbox-cloud-appliance monitor account create <account-name> [--password=<value>] [--email=<email>] [--role=viewer|admin|editor] [options]
+
+###### Options
+
+|Option | Description |
+|-------|-------------|
+| `-e`, `--email <email>`|Email address of the monitor account|
+| `-p`, `--password <value>`|Password of the monitor account|
+| `-r`, `--role <value>`|Role of the monitor account can be either of these three options: viewer/admin/editor. The default role is viewer.|
```

Comment:
Do we explain these roles somewhere? If we do, it may be useful to add a link, WDYT?

Reply:
We haven't explained it elsewhere as far as I could check, we could add it if there's more differentiation than the obvious to the roles. @morphis Is there a role definition here that we need to explain?

Reply:
Given that we drop these soon I don't see a strong need for that.